specification/cognitiveservices/OpenAI.Inference/models/completions/extensions_onyourdata_azure_cognitive_search_options.tsp

```diff
@@ -71,35 +81,33 @@ model AzureCognitiveSearchChatExtensionConfiguration {
   @projectedName("csharp", "SearchEndpoint")
   endpoint: url;
 
-  @doc("The API admin key to use with the specified Azure Cognitive Search endpoint.")
-  @projectedName("csharp", "SearchKey")
-  key: string;
-
   @doc("The name of the index to use as available in the referenced Azure Cognitive Search resource.")
   indexName: string;
 
   @doc("Customized field mapping behavior to use when interacting with the search index.")
   @projectedName("csharp", "FieldMappingOptions")
   fieldsMapping?: AzureCognitiveSearchIndexFieldMappingOptions;
 
-  #suppress "@azure-tools/typespec-azure-core/casing-style" "this represents the case-sensitive wire format"
-  @doc("The configured top number of documents to feature for the configured query.")
-  @projectedName("csharp", "DocumentCount")
-  topNDocuments?: int32;
-
   @doc("The query type to use with Azure Cognitive Search.")
   queryType?: AzureCognitiveSearchQueryType;
 
-  @doc("Whether queries should be restricted to use of indexed data.")
-  @projectedName("csharp", "ShouldRestrictResultScope")
-  inScope?: boolean;
-
   @doc("The additional semantic configuration for the query.")
   semanticConfiguration?: string;
 
-  @doc("When using embeddings for search, specifies the resource URL from which embeddings should be retrieved.")
+  @doc("Search filter.")
+  filter?: string;
+
+  @doc("When using embeddings for search, specifies the resource endpoint URL from which embeddings should be retrieved. It should be in the format of format https://YOUR_RESOURCE_NAME.openai.azure.com/openai/deployments/YOUR_DEPLOYMENT_NAME/embeddings?api-version={api-version}.")
+  @added(ServiceApiVersions.v2023_08_01_Preview)
+  @removed(ServiceApiVersions.v2023_12_01_Preview)
   embeddingEndpoint?: url;
 
+  @added(ServiceApiVersions.v2023_08_01_Preview)
+  @removed(ServiceApiVersions.v2023_12_01_Preview)
```

Comment:
We have both added and removed. Could be a pasting error, just FYI.

Reply:
The `@added` is redundant with the containing model having the same label; I'll remove those where I see them. Doesn't hurt anything but also very unnecessary.

Reply:
Actually, I take that back -- I'm not sure if it's a compiler quirk, but if I remove the `@added` I get:

> 'Azure.OpenAI.AzureCognitiveSearchChatExtensionConfiguration' was added on version 'v2023_08_01_Preview' but contains type 'Azure.OpenAI.AzureCognitiveSearchChatExtensionConfiguration.embeddingKey' added in version 'v2022_12_01'.TypeSpec(@typespec/versioning/incompatible-versioned-reference)